app/models/team.rb

```diff
@@ -30,6 +30,7 @@ class Team < ApplicationRecord
   belongs_to :team_captain, class_name: 'User'
   accepts_nested_attributes_for :user_invites
   validates :team_name, :affiliation, presence: true, length: { maximum: 255 }, obscenity: true
+  validates_presence_of :team_location, :unless => lambda { !Game.instance.location_required? }
```

Comment:
No need for the double negative here, `validates_presence_of :team_location, if: -> { Game.instance.location_required? }`